nilearn/datasets/atlas.py

```diff
@@ -268,37 +268,47 @@ def fetch_atlas_destrieux_2009(lateralized=True, data_dir=None, url=None,
     return Bunch(**params)
 
 
-def fetch_atlas_harvard_oxford(atlas_name, data_dir=None,
-                               symmetric_split=False,
-                               resume=True, verbose=1):
-    """Load Harvard-Oxford parcellations from FSL.
+def fetch_atlases_fsl(atlas_source, atlas_name, 
```

Comment:
This is not backward compatible. Users relying on `fetch_atlas_harvard_oxford` will have their code broken by this change. 
If we decide to combine harvard-oxford and Juelich fetchers in a new function `fetch_atlases_fsl`, we will need to run a deprecation cycle on `fetch_atlas_harvard_oxford` first. 
One way to do that would be to keep  `fetch_atlas_harvard_oxford` as a wrapper around `fetch_atlases_fsl` and gives a deprecation warning saying that `fetch_atlas_harvard_oxford` is deprecated, will be removed in release 0.9, and that the new usage is to rely on `fetch_atlases_fsl` with `atlas_name` set to `Harvard Oxford`.